### DISCUSSION TITLE
Custom theme not working

### DISCUSSION THREAD

**Discussion by forerosantiago** [upptime/upptime#448] [Q&A]
Hello, I just created the [dracula theme](https://github.com/forerosantiago/upptime/blob/master/assets/dracula.css) but I cannot apply it to [my site](https://status.forero.xyz/), apparently my css isn't being served as shown [here](https://upptime.js.org/docs/configuration/#theme), I'd be really happy if anyone could check my [upptime repo](https://github.com/forerosantiago/upptime) and see what's wrong or if this is a bug.

Thanks!

**Comment by forerosantiago** [comment]
I decided to upload my css file directly to my gh-pages branch. There is definitely something not working correctly.

**Comment by MatadorProBr** [comment]
https://github.com/forerosantiago/upptime/blob/c4a4ea540eded5f0c2324a2abebbece378c0fc92/.upptimerc.yml#L23
Maybe it's because you didn't remove the "#" on the "themeUrl", try remove the "#" and see if works

Also maybe use https://raw.githubusercontent.com/forerosantiago/upptime/master/assets/dracula.css instead of https://status.forero.xyz/dracula.css, because the https://status.forero.xyz/dracula.css doesn't work from what I'm seeing
